### PR TITLE
Fix Vertical scroller on empty screen

### PR DESCRIPTION
**Pull request by talentdev0807**
#2689
This bug comes from adding `Become a member` header
Originally, main body height is 100%, it is natural to have scroll when we add it on top screen
So, I made `Become a member` header as sticky header

### CODE REVIEW

**Comment by singulart** [comment]
Fixes https://github.com/Joystream/pioneer/pull/2958

**Comment by singulart** [comment]
Fixes https://github.com/Joystream/pioneer/issues/2689

File: packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx [hunk 1]
```diff
@@ -79,46 +79,57 @@ export const OnBoardingOverlay = () => {
...
+const TopSpace = styled.div`
+  width: 100%;
+  height: 90px;
+`
+
 const MainWrapper = styled.div`
-  position: relative;
+  position: fixed;
+  top: 0;
+  width: 100%;
```
Comment (stepien4soft) [inline packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx:125]: Hi @talentdev0807, First of all thank you for this PR. Just one small fix has to be added - for now <ButtonPrimary>Join now </ButtonPrimary> isn’t visible. Two quick solution is to change width for a main wrapper or set up padding on Join now button

**Review by stepien4soft — CHANGES_REQUESTED** [review]
Hi @talentdev0807, First of all thank you for this PR. Just one small fix has to be added - for now <ButtonPrimary>Join now </ButtonPrimary> isn’t visible. Two quick solution is to change width for a main wrapper or set up padding on Join now button

**Comment by stepien4soft** [comment]
Approved! Thanks for that

**Review by stepien4soft — APPROVED** [review]

**Pull request merged**